test/unit/spec/signaling/v2/twilioconnectiontransport.js

```diff
@@ -12,8 +12,40 @@ const TwilioError = require('../../../../../lib/util/twilioerror');
 const { combinations } = require('../../../../lib/util');
 
 describe('TwilioConnectionTransport', () => {
-  combinations([[true, false], [true, false], [true, false]]).forEach(([networkQuality, dominantSpeaker, automaticSubscription]) => {
-    describe(`constructor, called with .networkQuality flag ${networkQuality ? 'enabled' : 'disabled'}, .dominantSpeaker flag ${dominantSpeaker ? 'enabled' : 'disabled'} and .automaticSubscription flag ${automaticSubscription ? 'enabled' : 'disabled'}`, () => {
+  combinations([
+    [true, false],
```

Comment:
not sure if its worth trying various combinations of networkQuality, dominentSpeaker and all such values. maybe breaking up this test would make it more readable. Not a big concern though, as its just a unit test.

Reply:
@makarandp0 It helps to be thorough in unit tests, since integration tests are more costly.